litestar/cli/_utils.py

```diff
@@ -120,6 +121,8 @@ def from_env(cls, app_path: str | None, app_dir: Path | None = None) -> Litestar
 
             dotenv.load_dotenv()
         app_path = app_path or getenv("LITESTAR_APP")
+        if app_path and getenv("LITESTAR_APP") is None:
+            os.environ["LITESTAR_APP"] = app_path
```

Comment:
```suggestion
        app_path = os.environ["LITESTAR_APP"] = app_path or getenv("LITESTAR_APP")
```

No need for the check really. The value either comes from the env, in which case we can override it with its current value without changing anything, or it doesn't come from the env, in which case we want to set it.